expression/collation_test.go

```diff
@@ -29,8 +29,6 @@ import (
 )
 
 func TestCompareString(t *testing.T) {
-	t.Parallel()
```

Comment:
This is covered by #29276 . Please revert or rebase on #29276 .  I hope #29726 be merged soon.